pkg/loggers/logrus/logrus.go

```diff
@@ -92,6 +92,25 @@ func (l *logger) WithFields(fields chassis.Fields) chassis.Logger {
 	return newLogger(l.entry.WithFields(logrusFields))
 }
 
+// Implement `log.Logger` for `envoyproxy/go-control-plane/pkg/cache/v3`
+// go-control-plane has it's own logger interface that needs to be implemented for logging
+// to work correctly
+func (l *logger) Debugf(msg string, args ...any) {
```

Comment:
Probably should note in the doc comment on each function that you're dropping the `args` on the floor. Or better yet add them as fields before calling the subsequent write function. You should be able to call `WithFields()` to add them.

Reply:
It's def not clean. :(

Addin `WithFields()` is a good idea.

Reply:
Looks like `WithFields` is incompatible. It's expecting a `map[string]interface{}` and args is an `any`. I'm gonna add a comment to move on.